interpreter/cling/lib/Interpreter/LookupHelper.cpp

```diff
@@ -1535,11 +1522,27 @@ namespace cling {
        // Lookup failed.
        return 0;
     }
-    return functionSelector(foundDC,objectIsConst,GivenArgs,
-                            Result,
-                            FuncNameInfo,
-                            FuncTemplateArgs,
-                            Context, P, S, diagOnOff);
+
+    //  Suppress printing of diagnostics (we cannot simply ignore them
+    // since then it is not possible the check for errors programmatically
+    // either)
+
```

Comment:
```suggestion
    // Suppress printing of diagnostics (we cannot simply ignore them
    // since then it is not possible the check for errors programmatically
    // either).
```